project/metals.sbt

```diff
@@ -0,0 +1,4 @@
+// DO NOT EDIT! This file is auto-generated.
+// This file enables sbt-bloop to create bloop config files.
+
+addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "1.4.0-RC1-69-693de22a")
```

Comment:
This file shouldn't be here, can you add it to `.gitignore`

Reply:
Sorry about that!